scripts/perf-build-scenario

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+set -e
+
+
+if [ $# != 2 ]; then
+    echo "Usage: perf-build-scenario <scenario> <imagetag>"
+    exit 1
+fi
+
+SCENARIO="$1"
+shift
+TAG="$1"
```

Comment:
I think you mean "$2", but no reason not to just do:

```suggestion
TAG="ddtracepy/perf-${1}"
```

I think we can just skip the need to provide an image tag name

Reply:
I'm using `shift` here in case down the line we want to pass in command line arguments to docker at the end of this script's args. Down to go with using `${2}` instead.

However, we don't want to provide our own image tag name. That'll depend on how it's deployed and being run. For instance, this tag would include the registry if it's being pushed as it is in https://github.com/DataDog/dd-trace-py/blob/master/scripts/perf-build-push.

Reply:
oh, right, sorry I missed shift